ccViewer/ccViewerApplication.cpp

```diff
@@ -24,13 +24,22 @@
 #include "ccviewer.h"
 #include "ccViewerApplication.h"
 
-
+/*
+ * Initialise the ccViewerApp application
+ * this is version 1.40 alpha, it implements ccApplicationBase
+ * @param reference to argument
```

Comment:
the format is '\param' then you have to add the name of the argument
`\param argc number of arguments`